cmd/podman/system/service_abi.go

```diff
@@ -72,7 +72,7 @@ func restService(opts entities.ServiceOptions, flags *pflag.FlagSet, cfg *entiti
 	}
 
 	infra.StartWatcher(rt)
-	server, err := api.NewServerWithSettings(rt, opts.Timeout, listener)
+	server, err := api.NewServerWithSettings(rt, opts.Timeout, listener, opts.CorsHeaders)
```

Comment:
Should we change the API to just pass in opts?

Reply:
I thought about it and I think so. It will be easier.